.github/scality-storage-deployment/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  s3:
+    profiles: ['s3']
+    image: ${CLOUDSERVER_IMAGE}
+    network_mode: host
+    environment:
+      S3_CONFIG_FILE: /conf/config.json
+    command: /bin/sh -c "yarn run mem_backend > /logs/s3/s3.log 2>&1"
```

Comment:
Do you plan to eventually have a more complete S3 service (vault, metadata) ?

Reply:
I do not know yet, if the need arises then yes. Right now, the interaction is just with S3, adding MD and Vault won't change much.
We might add Vault for dynamic provisioning depending on strategy we decide to use for dynamic provisioning of PVs. But I do not see any value yet in adding MD, cuz CSI doesn't interact with MD directly.